app/charts/chart-loading-wrapper.tsx

```diff
@@ -86,12 +100,12 @@ export const ChartLoadingWrapper = <
       >
         <A11yTable
           title={title}
-          observations={observations.data}
+          observations={observations}
           dimensions={dimensions}
           measures={measures}
         />
         {React.createElement(Component, {
-          data: observations.data,
+          data: observations,
```

Comment:
If I am not mistaken, the Component here is loaded async; we could "attach" the getMetadata function a static method to it, so that it's loaded async.